docs/source/tutorials/Submission_to_NCBI.md

```diff
@@ -0,0 +1,38 @@
+# Bacterial genome submission to NCBI with VIGAS
+
+In the NVI VIGAS system it is possible to submit one or multiple genome sequences from isolates to the NCBI database so that you can accession numbers for these and the genome becomes public. But before you start with the submission in VIGAS, you first need to create a BIOPROJECT (in case it is not already existing) and you need to obtain BIOSAMPLE IDs. It is not possible to submit your genomes without these two items to the NCBI database. But don't worry. The process is very easy to follow and do. Things are automatically saved when you go to the next submission step at NCBI. So you can leave the process and come back the next day to continue, and it will do so at the step where you left.
+
```

Comment:
...database so that you can _get_ accession numbers....

Insert inline links to bioproject and biosample definitions/help pages if possible.

Reply:
Added the word "get" and added links to the general biosample and bioproject pages from NCBI